docs/source/metrics.rst

```diff
@@ -39,6 +39,13 @@ Metrics
 .. autoclass:: DiceMetric
     :members:
 
+`Mean IoU`
+-----------
+.. autofunction:: compute_meaniou
+
+.. autoclass:: IoUMetric
```

Comment:
To be consistent with the function, the metric class name may be `MeanIoU`?

Thanks.

Reply:
Okay, I'll do this